core/templates/core/community-update.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'base.html' %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+{% block extra_css %}
+<link rel="stylesheet" href="{% static 'core/css/style.css' %}">
+{% endblock %}
+
+{% block content %}
+<div class="container">
+    <div class="d-flex justify-content-center text-light">
+        <div class="mx-auto col-10 col-md-8 col-lg-8 mt-lg-3">
+            <div class="card bg-dark shadow-sm mb-3 text-light">
+                <div class="card-header">
+                    <h3 class="h3">Update Community: {{ community.name }}</h3>
+                </div>
+                <div class="card-body">
+                    <form method="post">
+                        {% csrf_token %}
+                        {% if form.non_field_errors %}
+                        <div class="alert alert-danger" role="alert">
+                            {% for err in form.non_field_errors %}
+                            <p class="form-error">{{ err }}</p>
+                            {% endfor %}
+                        </div>
+                        {% endif %}
+
+                        <div class="form-group">
+                            {{ form.name|as_crispy_field }}
+                        </div>
+
+                        <div class="form-group privacy-options">
+                            <label>Privacy Settings</label>
+                            {{ form.privacy|as_crispy_field }}
+                        </div>
+
+                        <div class="form-group">
+                            {{ form.is_18_plus|as_crispy_field }}
+                        </div>
+
+                        <button class="btn btn-sm btn-success" type="submit">Update Community</button>
+                    </form>
+                    {% if form.errors %}
+                    <div class="alert alert-danger mt-3">
+                        <ul>
+                            {% for field, errors in form.errors.items %}
```

Comment:
Isn't it a duplicate?
- non field errors at start of form display non field errors
- field errors should be displayed near the field no? If not, then core/templates/core/community-create.html is missing error display.